### DISCUSSION TITLE
Rounded slider and full height range

### DISCUSSION THREAD

**Discussion by ailchenkoDynamo** [radix-ui/primitives#1278] [Help]
Hello. Can somebody help me pls with my styles. I need the red color to stay within the "Thumb" limits. Now I have a left space when "Thumb" has start position and have problem on end "Thumb" position

[Codesandbox](https://codesandbox.io/s/practical-water-645oxl?file=/styles.css:1271-1274)

**Comment by benoitgrelard** [comment]
Hi @ailchenkoDynamo,

Unfortunately I am not sure what you are trying to achieve is possible currently.
This is due to how we calculate the offset for how we represent the value accurately on the range.

I'm not sure it would be possible to achieve without "lying" about the representation of the value.

**Reply by ailchenkoDynamo** [reply]
@benoitgrelard Thanks for your answer. I resolve my problem with custom range component with parse and recalculate right position. 

[Codesandbox](https://codesandbox.io/s/practical-water-645oxl?file=/App.js)

**Reply by benoitgrelard** [reply]
Nice!
